tests/container_tests.rs

```diff
@@ -407,7 +408,81 @@ async fn container_exec() {
         .await;
     assert!(exec_result.is_ok());
     let exec = exec_result.unwrap();
-    let mut exec_stream = exec.start(&opts).await.unwrap();
+    let mut exec_stream = exec.start(&opts).await.unwrap().unwrap();
+    let chunk = exec_stream.next().await;
+    assert!(chunk.is_some());
+    match chunk.unwrap() {
+        Ok(TtyChunk::StdOut(chunk)) => {
+            let testfile_content = String::from_utf8_lossy(&chunk);
+            assert_eq!(testfile_content, "1234\n");
+        }
+        Ok(chunk) => {
+            let fd = match chunk {
+                TtyChunk::StdIn(_) => "stdin",
+                TtyChunk::StdOut(_) => "stdOut",
+                TtyChunk::StdErr(_) => "stderr",
+            };
+            let chunk = String::from_utf8_lossy(&chunk);
+            eprintln!("invalid chunk, fd: {fd}, content: `{chunk:?}`");
+            std::process::exit(1);
+        }
+        chunk => {
+            eprintln!("invalid chunk {chunk:?}");
+            std::process::exit(1);
+        }
+    }
+
+    cleanup_container(&podman, container_name).await;
+}
+
+#[tokio::test]
+async fn container_exec_detach() {
+    let podman = init_runtime();
+
+    let container_name = "test-exec-container";
```

Comment:
Container name should be unique per test case so it doesn't cause problems when running tests simultaneously, one possibility is for example `test-exec-detach-container`

Reply:
Fixed in bamboocleaver@3df0c38a440de8272b64b576d5285e065644e6e6